connection_tracker/README.md

```diff
@@ -0,0 +1,41 @@
+# eBPF For Windows Demo - Connection Tracking
```

Comment:
I think we should add a link to this md file in the top level readme also, which lists all the demos in this repo.